internal/hosts/hosts.go

```diff
@@ -97,6 +97,22 @@ func (n *Host) GetSAPSystems() (map[string]*sapsystem.SAPSystem, error) {
 	return systems, nil
 }
 
+func (n *Host) GetSAPSystemsList() (sapsystem.SAPSystemsList, error) {
+	systems, err := sapsystem.Load(n.client, n.Name())
```

Comment:
I don't know if it's a good idea to do this...
Cannot we simply use the tag value and split the string by the comma entry?